ui/shared/components/panel/variants/VariantIndividuals.jsx

```diff
@@ -377,10 +358,106 @@ const Genotype = React.memo(({ variant, individual, isCompoundHet, genesById })
           {filters}
         </small>
       )}
+    </div>
+  )
+}
+
+GenotypeQuality.propTypes = {
+  genotype: PropTypes.object,
+  variant: PropTypes.object,
+}
+
+const getWarningsForGenotypes = (genotypes, variant, isHemiX, warnings) => {
+  const formattedWarnings = []
+
+  const hasDifferentNumAlt = genotypes.some(genotype => genotype.numAlt !== genotypes[0].numAlt)
+  if (hasDifferentNumAlt) {
+    formattedWarnings.push('Genotypes differ across sample types')
+    formattedWarnings.push(
+      genotypes.map(genotype => (
+        <div key={genotype.sampleType || genotype.sampleId}>
+          {genotype.sampleType || genotype.sampleId}
+          :
+          <Alleles genotype={genotype} variant={variant} isHemiX={isHemiX} />
+        </div>
+      )),
+    )
+  }
+
+  if (Object.keys(warnings).length > 0) {
+    Object.entries(warnings).forEach(([warning, sampleTypes]) => {
+      formattedWarnings.push(`${warning} (${Array.from(sampleTypes).join(', ')})`)
+    })
+  }
+
+  return formattedWarnings.length > 0 && (
+    <div>
+      {formattedWarnings.map(warn => (
+        <div key={warn}>{warn}</div>
+      ))}
+    </div>
+  )
+}
+
+const Genotype = React.memo(({ variant, individual, isCompoundHet, genesById }) => {
+  const individualGenotypes = variant.genotypes[individual.individualGuid]
+  if (!individualGenotypes) {
+    return null
+  }
+  const genotypes = (Array.isArray(individualGenotypes) ? individualGenotypes : [individualGenotypes]).sort(
+    (a, b) => SAMPLE_TYPE_DISPLAY_ORDER.indexOf(a.sampleType) - SAMPLE_TYPE_DISPLAY_ORDER.indexOf(b.sampleType),
+  )
+
+  if (genotypes.every(genotype => !isCalled(genotype.cn) && !isCalled(genotype.numAlt))) {
+    return <b>NO CALL</b>
```

Comment:
Do you think there will be a problem with the way I'm handling this NO CALL code path?